.github/workflows/packaging.yml

```diff
@@ -78,50 +88,202 @@ jobs:
           if-no-files-found: error
 
       - name: Upload AppImage
         if: matrix.platform.os == 'linux'
         uses: actions/upload-artifact@v4
         with:
-          name: "nomos-node_${{ env.VERSION }}_${{ matrix.platform.arch }}.AppImage"
-          path: "target/${{ matrix.platform.target_triple }}/debug/bundle/appimage/nomos-node_${{ env.VERSION }}_amd64.AppImage"
+          name: "${{ matrix.platform.arch }}_appimage"
+          path: "target/${{ matrix.platform.target_triple }}/release/bundle/appimage/nomos-node_${{ env.VERSION }}_amd64.AppImage"
           if-no-files-found: error
 
       - name: Upload DMG
         if: matrix.platform.os == 'macos'
         uses: actions/upload-artifact@v4
         with:
-          name: "nomos-node_${{ env.VERSION }}_${{ matrix.platform.arch }}.dmg"
-          path: "target/${{ matrix.platform.target_triple }}/debug/bundle/dmg/nomos-node_${{ env.VERSION }}_${{ matrix.platform.arch }}.dmg"
+          name: "${{ matrix.platform.arch }}_dmg"
+          path: "target/${{ matrix.platform.target_triple }}/release/bundle/dmg/nomos-node_${{ env.VERSION }}_${{ matrix.platform.arch }}.dmg"
           if-no-files-found: error
 
       - name: Upload MSI
         if: matrix.platform.os == 'windows'
         uses: actions/upload-artifact@v4
         with:
-          name: "nomos-node_${{ env.VERSION }}_${{ matrix.platform.arch }}.msi"
-          path: "target/${{ matrix.platform.target_triple }}/debug/bundle/msi/nomos-node_${{ env.VERSION }}_amd64.msi"
+          name: "${{ matrix.platform.arch }}_msi"
+          path: "target/${{ matrix.platform.target_triple }}/release/bundle/msi/nomos-node_${{ env.VERSION }}_amd64.msi"
           if-no-files-found: error
+
+  create-release:
+    name: "Create release"
+    needs:
+      - setup
+      - bundle
+    runs-on: ubuntu-latest
+    env:
+      VERSION: ${{ needs.setup.outputs.version }}
+    outputs:
+      upload_url: ${{ steps.create_release.outputs.upload_url }}
+    steps:
+      - name: Create Release
+        uses: actions/create-release@v1
+        id: create_release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ env.VERSION }}
+          release_name: Nomos Node ${{ env.VERSION }}
+          body: |
+            This is a release of Nomos Node ${{ env.VERSION }}.
+            ## Changelog
+            - feature(X): new feature
+            - fix(Y): bug description
+            - feature: performance improvement on Z
+            ## Checklist
+            Before publishing please ensure:
+            - [ ] Description is complete
+            - [ ] Changelog is correct
+            - [ ] Assets for all platforms exist
+            - [ ] Pre-release is checked if necessary
+            - [ ] Remove this checklist before publishing the release.
+          draft: true
+          prerelease: true
```

Comment:
This checklist is for a final check just before removing the `prerelease` badge?

Reply:
Actually, that'd be the `draft`; but yes.
`prerelease` should be set to `false`, I believe, as it's intended to mark a `beta` or `alpha` release (or the likes of).